docs/pages/faq.mdx

```diff
@@ -15,32 +15,25 @@ Fortune 500 companies. It has been through several security audits from
 nationally recognized technology security companies, so we are comfortable with
 the stability of Teleport from a security perspective.
 
-## Can Teleport be deployed in agentless mode?
-
-Yes. All Teleport services support agentless mode, where the service proxies
-traffic to an upstream infrastructure resource not available on `localhost`.
+## Can I connect to nodes behind a firewall?
 
-With Teleport in agentless mode, you can easily control access to SSH servers,
-Kubernetes clusters, desktops, databases, and internal applications without
-running any additional software on your servers. Agentless mode supports session
-recordings and audit logs for deep understanding into user behavior.
+Yes, Teleport supports reverse SSH tunnels out of the box. To configure
+behind-firewall clusters, see [Configure Trusted Clusters](admin-guides/management/admin/trustedclusters.mdx).
 
-For capabilities such as kernel-level logging and user provisioning, we
-recommend Teleport as a drop in replacement for OpenSSH. Since Teleport replaces
-the OpenSSH agent while preserving OpenSSH's functionality, you get more
-functionality without a net addition of an agent on your system.
+## How is Teleport's Community Edition different from Enterprise?
 
-## Can I use OpenSSH with a Teleport cluster?
+Teleport provides three editions:
 
-Yes, this question comes up often and is related to the previous one. Take a
-look at [Using OpenSSH Guide](enroll-resources/server-access/openssh/openssh-agentless.mdx).
+- Teleport Enterprise
+- Teleport Enterprise Cloud
```

Comment:
@garofalof Should we say:
- We have two editions, Teleport Enterprise and Teleport Community Edition
- We have three editions, Teleport Enterprise (Cloud), Teleport Enterprise (Self-Hosted), and Teleport Community Edition

Reply:
I'd vote for the latter, the parenthetical "(Cloud)" qualifier drives me crazy.

Reply:
I tend to agree. I think the origin here is that Marketing wanted Enterprise to be positioned as a singular product, which led to the parenthetical qualifiers, but I'm also not a fan. Let me check if we can move to just stating the 3 editions without parentheses.